dali/kernels/erase/erase_gpu.h

```diff
@@ -432,7 +437,7 @@ struct EraseGpu {
   void Run(KernelContext &ctx,
            OutListGPU<T, ndim> &out,
            const InListGPU<T, ndim> &in,
-           const InListGPU<ibox<ndim>, 1> &erased_regions,
+           const std::vector<std::vector<ibox<ndim>>> &erased_regions,
```

Comment:
Likewise - please revert to GPU-based data.

Reply:
Why? It's an input argument, it will be supplied on CPU and I think it's kernel's responsibility to allocate GPU memory and move it there.